packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -29,8 +29,12 @@ class SemanticsService {
   ///
   /// For example a camera application can use this method to make accessibility
   /// announcements regarding objects in the viewfinder.
-  static Future<void> announce(String message, TextDirection textDirection) async {
-    final AnnounceSemanticsEvent event = AnnounceSemanticsEvent(message, textDirection);
+  ///
+  /// The third argument adds an optional setting for flutter web.
+  /// It determines the politeness setting of aria live.
+  /// The default mode is 'assertive'.
+  static Future<void> announce(String message, TextDirection textDirection, [AriaLivePolitenessSetting ariaLivePolitenessSetting = AriaLivePolitenessSetting.assertive]) async {
```

Comment:
Consider using an optional named parameter, rather than optional positional parameter. This would better scale to future  expansions of the API. For example, if we decide to add a fourth parameter, the caller would be forced to set `ariaLivePolitenessSetting` even if they are content with the default.